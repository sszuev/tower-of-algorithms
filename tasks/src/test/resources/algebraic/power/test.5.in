1.000001
1000000
