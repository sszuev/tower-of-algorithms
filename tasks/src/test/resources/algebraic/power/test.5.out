2.7182804691